04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -13,7 +13,7 @@ output:
 
 In this example, we use weighted gene co-expression network analysis (WGCNA) to identify co-expressed gene modules [@Langfelder2008]. 
 WGCNA uses a series of correlations to identify sets of genes that are expressed together in your data set. 
-This is a fairly intuitive approach to gene network analysis which can aid in interpretation of microarray & RNAseq data. 
+This is a fairly intuitive approach to gene network analysis which can aid in interpretation of microarray & rna-seq data. 
```

Comment:
```suggestion
This is a fairly intuitive approach to gene network analysis which can aid in interpretation of microarray & RNA-seq data. 
```

Reply:
I changed it to `rna-seq` because for some silly reason I thought that was what we had elsewhere, but I think we have `RNA-seq` most places. But this is something pertinent to #289 or maybe it deserves its own issue.